sns_ik_lib/test/sns_ik_math_utils_test.cpp

```diff
@@ -22,8 +22,8 @@
 #include <Eigen/Dense>
 #include <ros/console.h>
 
-#include <sns_ik/rng_utilities.hpp>
-#include <sns_ik/sns_ik_math_utils.hpp>
+#include "rng_utilities.hpp"
+#include "../src/sns_ik_math_utils.hpp"
```

Comment:
bah to relative paths, but I can't think of an elegant way around it off the top of my head.

Reply:
I think its possibly an indicator that we should consider adding a subset of math utilities to the public API?

Reply:
I think that this is fixed now. I moved the utilities to their own private library, which both the test and main libraries link against. Let me know if I managed the CMakeLists correctly for it.